CONTRIBUTING.md

```diff
@@ -34,6 +34,9 @@ Issues labeled [help wanted](https://github.com/medic/cht-core/labels/Help%20wan
 5. Your PR will be reviewed by one of the repository's maintainers. Most PRs have at least one change requested before they're merged so don't be offended if your change doesn't get accepted on the first try!
 
 ### Improving our documentation
+
+**Note:** [medic-docs](https://github.com/medic/cht-docs) does not involve release management and acceptance testing. Help us maintain the quality of our documentation by submitting a pull request (PR) with any suggested changes.
```

Comment:
We renamed this repo from medic-docs -> cht-docs so we should fix this at the same time.
```suggestion
**Note:** [cht-docs](https://github.com/medic/cht-docs) does not involve release management and acceptance testing. Help us maintain the quality of our documentation by submitting a pull request (PR) with any suggested changes.
```